src/D2L.CodeStyle.Analyzers/ApiUsage/ImmutableGenericAttributeAnalyzer.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Immutable;
+using System.Linq;
+using D2L.CodeStyle.Analyzers.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace D2L.CodeStyle.Analyzers.ApiUsage {

```

Comment:
Could I bug you to throw this under `D2L.CodeStyle.Analyzers.Immutability` instead?